client/dist/bundle.js.LICENSE.txt

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This file should not be committed. Delete/ add to gitignore like you did bundle.js.